docs/development/managing-users/index.md

```diff
@@ -1,16 +1,16 @@
 # Managing users
 
-To manage users you will need an admin token and connection to the REANA-Server component:
+To manage users you will need to obtain administration credentials:
 
 ```console
-$ kubectl exec -i -t reana-server-aaaa-bbbb /bin/bash
-root@reana-server-aaaa-bbbb:/code# read -s REANA_ADMIN_ACCESS_TOKEN
+$ export KUBECTL=~/mycluster/config
```

Comment:
Isn't it `KUBECONFIG`? I might be wrong though. I've never seen `KUBECTL` env var.